neuralpredictors/layers/shifters/static_affine.py

```diff
@@ -0,0 +1,45 @@
+from .base import Shifter
+
+import torch
+from torch import nn
+from torch.nn import ModuleDict
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class StaticAffine2dShifter(Shifter, ModuleDict):
```

Comment:
I'd prefer that we keep consistency in the order. In the `mlp.py` the `MLP` class came before `MLPShifter`. In the same spirit, I would prefer to see `StaticAffine2d` before `StaticAffine2dShifter` defined.